server/controllers/MissionController.js

```diff
@@ -0,0 +1,22 @@
+const { getMission, updateMission } = require('../store/missions');
+const { createMissionUpdate } = require('../store/mission_updates');
+const { getVehicle, updateVehicleStatus } = require('../store/vehicles');
+
+
+const command = async (req, res) => {
+  const { user_id, mission_id, command} = req.query;
+  const mission = await getMission(mission_id);
+  const vehicle = await getVehicle(mission.vehicle_id);
+
+  if (user_id !== mission.user_id) res.sendStatus(401);
```

Comment:
Does `res.sendStatus()` also exit/returns? If not, we should `return` out of this function so it doesn't continue to do the mission/vehicle updates even when user is unauthorized.

Reply:
It doesn't exit/return. Pushing that now.